elements/customer_order_slip.php

```diff
@@ -10,7 +11,8 @@
 <html>
 <head>
     <title><?= t("Order #") . $order->getOrderID() ?></title>
-    <link href="<?= str_replace('/index.php/', '/' , \URL::to('/concrete/css/app.css')); ?>" rel="stylesheet" type="text/css" media="all">
+    <link href="<?= str_replace('/index.php/', '/', \URL::to('/concrete/css/cms.css')); ?>" rel="stylesheet"
```

Comment:
This assumes the use of Concrete V9+ and will break with V8 sites.  I'd suggest that we need to do:

`if (version_compare(\Config::get('concrete.version'), '9.0', '>=')) {`
....

or similar.